packages/icons/src/HarnessIcons.ts

```diff
@@ -251,6 +251,7 @@ import Down from './down.svg'
 import EcrStepInverse from './ecr-step-inverse.svg'
 import EcrStep from './ecr-step.svg'
 import ElasticKubernetesService from './elastic-kubernetes-service.svg'
+import Elk from './elk.svg'
 import EmailInline from './email-inline.svg'
 import EmailStep from './email-step.svg'
```

Comment:
Do we also need to update the version in package.json ?

Reply:
yes. we do.